docs/makeup-modal/index.html

```diff
@@ -14,7 +14,8 @@
     <body>
         <main role="main">
             <h1>makeup-modal demo</h1>
-            <p>The dotted region below has been set to modal, using the modal() method.</p>
+            <p>The dotted region below will be set to modal on click of the "Modal" button, using the modal() method.</p>
```

Comment:
Is there a reason for changing this demo page, rather than just adding a specific demo page for hoist element? If the API is working then this particular page shouldn't require any changes.

Reply:
Also, for when creating a demo page for hoist-element, there is no need to mention anything about dialogs or traps. The demo should simply show an element being hoisted (probably executed after a button click). The idea being that hoist-element might also have use-cases outside of a dialog scenario.

Reply:
@austingardner Can you revert the changes to the modal demo? The changes have broken the original demo I had. Once reverted we can take a look and figure out a way to do a hoisted demo and a non-hoisted.